app/task_routes.py

```diff
@@ -0,0 +1,131 @@
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+import datetime
+import requests
+from app import token 
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_task(id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message": f"Task {id} is invalid"}, 400))
+
+    task = Task.query.get(id)
+
+    if not task:
+        abort(make_response({"message": f"Task {id} not found"}, 404))
+
+    return task
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    if request.method == "POST":
```

Comment:
No need to check if the request method is POST, since you specified on line 24 with the decorator that this route `create_task()` is for POST requests.